yaksh/models.py

```diff
@@ -251,6 +251,9 @@ class Profile(models.Model):
         default=pytz.utc.zone,
         choices=[(tz, tz) for tz in pytz.common_timezones]
     )
+    is_email_verified = models.BooleanField(default=False)
+    activation_key = models.CharField(max_length=40, blank=True, null=True)
```

Comment:
Is it necessary to hold onto the activation key in the database once the verification is done?